<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>523</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Python PWC Message Kit</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-10</y>
      <width>281</width>
      <height>61</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>801</width>
      <height>61</height>
     </size>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#ffa348;&quot;&gt;Python PWC Message Kit&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>291</width>
      <height>371</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QComboBox" name="x_filetype">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="statusTip">
      <string>File Type</string>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
     <property name="placeholderText">
      <string>File Type</string>
     </property>
    </widget>
    <widget class="QComboBox" name="x_company">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="statusTip">
      <string>Company Name</string>
     </property>
     <property name="placeholderText">
      <string>Company</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="x_loadid">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>160</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="statusTip">
      <string>Load ID</string>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>271</width>
       <height>101</height>
      </rect>
     </property>
     <property name="statusTip">
      <string>Input File</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPlainTextEdit" name="x_inputfile">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>251</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="statusTip">
       <string/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="backgroundVisible">
       <bool>false</bool>
      </property>
      <property name="centerOnScroll">
       <bool>false</bool>
      </property>
      <property name="placeholderText">
       <string>Select the file to upload...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_browse">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>70</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_execute">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>70</y>
        <width>20</width>
        <height>25</height>
       </rect>
      </property>
      <property name="statusTip">
       <string>Open selected file</string>
      </property>
      <property name="text">
       <string>#!</string>
      </property>
     </widget>
    </widget>
    <widget class="QComboBox" name="x_datatype">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="statusTip">
      <string>Data Type</string>
     </property>
     <property name="placeholderText">
      <string>Data Type</string>
     </property>
    </widget>
    <widget class="QComboBox" name="x_filesubtype">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>220</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <property name="statusTip">
      <string>File Sub Type</string>
     </property>
     <property name="placeholderText">
      <string>File Sub Type</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="x_originalfilename">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="statusTip">
      <string>Original File Name</string>
     </property>
     <property name="placeholderText">
      <string>Original File Name</string>
     </property>
    </widget>
    <widget class="QComboBox" name="x_rabbit_queue">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>271</width>
       <height>25</height>
      </rect>
     </property>
     <property name="statusTip">
      <string>Rabbit Queue Name</string>
     </property>
     <property name="placeholderText">
      <string>Rabbit Queue Name</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="x_dbname">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>250</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="statusTip">
      <string>DB Name</string>
     </property>
     <property name="placeholderText">
      <string>DB Name</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="x_foldername">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="statusTip">
      <string>Folder Name</string>
     </property>
     <property name="placeholderText">
      <string>Folder Name</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="x_bucketname">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>290</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="statusTip">
      <string>Bucket Name</string>
     </property>
     <property name="placeholderText">
      <string>Bucket Name</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_run">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>450</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTextBrowser" name="console">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>331</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clear">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>450</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>Clear the console</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuTheme">
     <property name="title">
      <string>Theme</string>
     </property>
     <addaction name="actionLight"/>
     <addaction name="actionDark"/>
     <addaction name="actionDefault"/>
    </widget>
    <addaction name="menuTheme"/>
    <addaction name="actionConfiguration"/>
    <addaction name="actionShortcuts"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>About</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>false</bool>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionConsole"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuView"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionConfiguration">
   <property name="text">
    <string>Configuration</string>
   </property>
  </action>
  <action name="actionShortcuts">
   <property name="text">
    <string>Shortcuts</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionConsole">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Console</string>
   </property>
  </action>
  <action name="actionLight">
   <property name="text">
    <string>Light</string>
   </property>
  </action>
  <action name="actionDark">
   <property name="text">
    <string>Dark</string>
   </property>
  </action>
  <action name="actionDefault">
   <property name="text">
    <string>Default</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>Information</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_clear</sender>
   <signal>released()</signal>
   <receiver>console</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>575</x>
     <y>477</y>
    </hint>
    <hint type="destinationlabel">
     <x>494</x>
     <y>445</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
